eng/pipelines/common/evaluate-default-paths.yml

```diff
@@ -21,6 +21,7 @@ jobs:
       - src/installer/*
       - src/mono/*
       - src/libraries/*
+      - src/tests/*
```

Comment:
Any reason changes to coreclr and mono runtimes don't trigger on changes to the tests? seems unexpected to me

Reply:
Hmm, I find that reasonable - if you change something in coreclr tests, I don't see why you'd need to re-evaluate the stability of coreclr runtime build, that's not affected by the tests at all (at least to my knowledge). I believe the same holds for mono. Please correct me if I misunderstood your comment.

Reply:
> Any reason changes to coreclr and mono runtimes don't trigger on changes to the tests? seems unexpected to me

That is the point of having a `runtimetests` partition. However, if you look at the `runtimetests` conditions, if someone changes the tests partition, it will run runtime tests for both Mono and CoreCLR runtime. 

The reason why I fixed this is because there were cases where a coreclr change (that didn't touch the runtime tests), cause mono tests to run on that PR and that is not desired.

Reply:
Didn't recall that subset. Github shows limited context. Thanks for explaining.